index.html

```diff
@@ -1182,7 +1190,7 @@
     <p>
       The <dfn>NFCMessage.url</dfn>
       property represents the <a>Web NFC Id</a> of a received
-      <a>Web NFC message</a>. When used in the <code><a>NFC.push</a>()</code>
+      <a>Web NFC message</a>. When used in the <code><a>NFCWriter.push</a>()</code>
```

Comment:
Nit: Consider writing something like NFCWriter.prototype.push (fine to do this in another patch)

Reply:
Sure. I hope that works with respec :-)